app/views/admin/induction_coordinators/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Induction coordinators</h1>
+    <table class="govuk-table">
+      <thead>
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Name</th>
+          <th scope="col" class="govuk-table__header">Email address</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @induction_coordinators.each do |induction_coordinator| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= govuk_link_to induction_coordinator.full_name, admin_edit_user_path(induction_coordinator) %></td>
```

Comment:
I'm surprised it's just `admin_edit_user_path`

Reply:
I think it's a helper method added in an earlier PR?

Reply:
Although, if we go for solution without much repetition (one controller to rule and edit all users), it will be this short without the need for a helper method.

Reply:
I thought really hard about the repetition and had a quick chat with Seb too. Our conclusion was that we not worry about it too much, these requirements are too uncertain for us to worry about optimisation. My gut feeling is that these endpoints will diverge more as we gather more requirements and it's often easier to go from separate endpoints to one shared endpoint than the other way round.

Reply:
Alright. I think it tends to be easier to go from one endpoint to many - cause you can start from copy-pasting the pages, controllers, and routes, and then delete things you don't want in each. 

But I think I'm convinced that there will be enough divergence between these pages that we should set up the skeleton separate flows for them.